optimize/pom.xml

```diff
@@ -691,6 +690,30 @@
       </snapshots>
       <layout>default</layout>
     </repository>
+
```

Comment:
❓ Why are we adding the bpm repo? I think I'm missing some context here

Reply:
The BPM repo is public and is where the license checker library lives. The previous repo for the license checker was not public and people could not pull down the dependency

Placing it in the bpm repo makes that checker available to download